sql/updates/hotfixes/master/9999_99_99_18010_hotfixes.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS `conversation_line`;
```

Comment:
This `DROP TABLE` command does not match `CREATE TABLE`.
Going to remedy this in my commit as well.